### PR TITLE
Remove caching code

### LINKED ISSUE #2895: Remove caching code
**Is your feature request related to a problem? Please describe.**

The caching is not used anymore after the mobile improvements. Let's remove the according code.

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by mariopino**
Closes #2895

**Description:**

This is a working in progress PR:

- Caching code has been removed from `src/vuex/store.js`. Everything works as expected in Lunie, including making a transaction and sign it using the extension.

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by faboweb** [comment]
@mariopino changelog missing ;)

**Comment by jbibla** [comment]
i love deleting code

-301 😄 😄

**Comment by faboweb** [comment]
@mariopino e2e tests fail for this PR

**Comment by mariopino** [comment]
> @mariopino e2e tests fail for this PR

Oki, I am going to check it!

File: package.json [hunk 1]
```diff
@@ -18,8 +18,8 @@
     "test:coverage-branch": "npm run test:unit -- --coverage && git diff origin/develop | diff-test-coverage -c ./coverage/lcov.info -t lcov --",
     "test:e2e:browserstack": "node tests/e2e/browserstack.runner.js -c  tests/e2e/browserstack.conf.js",
     "test:e2e:local": "kill-port 9515 && nightwatch --config ./tests/e2e/local-chrome.conf.js",
-    "test:e2e:build": "cross-env VUE_APP_E2E=true STARGATE=http://localhost:9070 RPC=http://localhost:26657 vue-cli-service build --mode production",
-    "test:e2e:serve": "kill-port 9080 && cross-env VUE_APP_E2E=true STARGATE=http://localhost:9070 RPC=http://localhost:26657 vue-cli-service serve --mode production --port 9080 --https",
+    "test:e2e:build": "cross-env VUE_APP_E2E=true VUE_APP_GRAPHQL_URL=https://backend.lunie.io/v1/graphql STARGATE=http://localhost:9070 RPC=http://localhost:26657 vue-cli-service build --mode production",
```
Comment (faboweb) [inline package.json]: This should be `VUE_APP_GRAPHQL_URL=http://localhost:8080/v1/graphql` as the e2e tests should be run in an isolated setup. Imagine testing delegations.

Comment (mariopino) [reply package.json]: But no api v1 is deployed with e2e CI setup at por 8080 isn't? When I change to VUE_APP_GRAPHQL_URL=http://localhost:8080/v1/graphql tests fail again :-(

Comment (faboweb) [reply package.json]: You are correct. happy with your solution then. Feel free to revert.

**Pull request merged**